extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -256,6 +256,33 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
   }
 
+  /**
+   * @private
+   * @return {Element}
+   */
+  maybeConvertCtaLayerToPageOutlink_() {
+    const ctaLayerEl = this.element.querySelector('amp-story-cta-layer');
+    if (!ctaLayerEl) {
+      return;
+    }
+
+    const anchorSet = ctaLayerEl.querySelectorAll('a');
+    if (anchorSet.length !== 1 || !anchorSet[0].getAttribute('href')) {
```

Comment:
In the case that there are more than one anchor elements, the elements should trigger a tooltip like page links outside of the cta layer.

Reply:
Done in page-advancement.